controllers/constant/odlm.go

```diff
@@ -431,6 +611,12 @@ spec:
     installPlanApproval: {{ .ApprovalMode }}
     sourceName: {{ .CatalogSourceName }}
     sourceNamespace: {{ .CatalogSourceNs }}
+  - channel: stable
```

Comment:
Could we move this part into `const CSV3OperandRegistry`?

Reply:
`cloud-native-postgresql` already added in `const CSV3OperandRegistry`. I think EDB will support multi ns in the future, but if we prefer not to include them in saas at this time, I will remove it.

Reply:
Ah I see, sorry that I did not notice it.

For `CSV3SaaSOperandRegistry`, it really depends on whether EDB will be SaaS scenario.

For `CSV3OperandRegistry`, it already supports both on-prem single and multi instances installation, so the multi ns for EDB will be covered in this case.

Let's confirm with @horis233 about the SaaS

Reply:
We don't need to add it in SaaS until we get any further notification.